introduction/doc/index.mdx

```diff
@@ -1,5 +1,97 @@
+
+> React Aria ensures consistent behavior, no matter the UI.
+
+> React Aria provides accessibility and behavior according to WAI-ARIA Authoring Practices, including full screen reader and keyboard navigation support.
+
+Because of the agnostic nature of React Aria, the Design Tokens for styling are declared in [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).
+
+The components then use [Scss](https://sass-lang.com/) for their internal styles, relying on CSS Custom Properties for Tokens access.
+
+### Tokens
+
+The Tokens are split in different components:
+
+- `typography`: fonts definitions
+- `colors`: colors and variants
+- `spacing`: relative and absolute spaces
+- `elevation`: shadows effects
+- `border-radius`
+- `transitions`: easing and speed
+- `z-index`: z-axe indices
+
+All tokens are declared in dedicated `_*.scss` files in their dedicated tokens' components. They declare CSS Custom Properties, prefixed with `--aria-*`. They're then all exposed by the `all/all.scss` component global stylesheet.
+
+Your theme tokens are accessible by importing the theme stylesheet:
+
+```ts
+import '~/all/src/all.scss';
+```
+
+## Components
+
+Components can access the Tokens anywhere in their stylesheets as they're exposed at top CSS level `:root`.
+
+When styled with SCSS, components can use the [Sass `var` method](https://sass-lang.com/documentation/breaking-changes/css-vars) to read the Tokens values.
+
+```scss
+.button {
+  border-radius: var(--aria-border-radius-medium);
+  background: var(--aria-color-primary);
+  color: var(--aria-color-white);
+}
+```
+
+Then the styles are imported in the component using a CSS in React module interpolation:
+
+```ts
+import styles from './button.module.scss';
+
+export const Button => (
+  <button className={`${styles.button}`}>
+    // ...
+  </button>
+)
+```
+
+## Stories
+
+Stories are written in Storybook's [Component Story Format](https://backlight.dev/docs/component-story-format).
+
+The theme stylesheet containing the Tokens is injected in all stories files, thanks to the `story-layout` component:
+
+```jsx
+import '~/story-layout';
+import React from 'react';
+import { Button } from '../index';
+
+export const primary = () => <Button>Primary</Button>;
+```
+
+Stories for the components are located in their `stories/` folder.
+
+## Documentation
+
+Documentation pages are decorated by a React layout using the [@divriots/dockit-react](https://github.com/divriots/dockit-react) helpers. See the `mdx-layout` component.
+
+### Pages
```

Comment:
Maybe Pages and Documentation is redundant. Should probably be a single paragraph for documentation.